### ISSUE TITLE
USWDS-Site - Bug: Remove "navigation" text from aria labels for navigation roles

### ISSUE THREAD

**Issue by littleforest** [uswds/uswds-site#3059]
### Describe the bug

To avoid redundancy, the word "navigation" should be removed from examples as well as the USWDS site itself, as screen readers will read both the role and the contents of the label. Removing this redundancy in examples will also help users perpetuate the correct usage.

[Documentation reference](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#associated_wai-aria_roles_states_and_properties)

### Steps to reproduce the bug

Example 1: 
1. Go to any page of the [USWDS documentation](https://designsystem.digital.gov/)
3. Read the navigation with a screen reader, or open the rotor.
4. Note that the screen reader reads "Primary navigation navigation"
5. [Related code](https://github.com/uswds/uswds-site/blob/071993154b20b926bd1a3f81eb024c853d4de822/_includes/navbar.html#L16)
<img width="634" alt="Image" src="https://github.com/user-attachments/assets/e270b531-1770-407d-8bae-3119102d0d62" />

Example 2:
1. Copy the component code for the [Footer](https://designsystem.digital.gov/components/footer/) into an HTML file.
2. Read with a screen reader, or open the rotor.
3. Note that the screen reader reads "Footer navigation navigation"

<img width="634" alt="Image" src="https://github.com/user-attachments/assets/306c1207-cac2-47b4-b095-eb8fce720896" />

Other area of the site that may be affected:
- https://github.com/uswds/uswds-site/blob/071993154b20b926bd1a3f81eb024c853d4de822/_includes/next/next-prefooter.html#L37

### Expected Behavior

The screen reader should read: "Primary navigation" or "Footer navigation" without the redundant "navigation" word.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.

**Comment by amycole501** [comment]
@juliaelman this is related to https://github.com/uswds/uswds/issues/6301 so if this gets fixed the component can also be updated with the same fix. Happy to test with NVDA when ready.
